### PR TITLE
feat: Printing translation

### PR DESCRIPTION
**Pull request by zhanghuize-git**
### Description

add Printing translation and modify some window.print translation

### Motivation

readers in China need this content when he look for print knowledge

### CODE REVIEW

**Comment by yin1999** [comment]
Hey @OnkarRuikar. reviewdog created many duplicate suggestions here. Can you take a look :)

I'm not sure if this is caused by [the temporary commit](https://github.com/mdn/translated-content/blob/d9b0bf1d36ca49aa42b9e88547ee86718b56d994/.github/workflows/pr-check-lint_content.yml#L36-L40).

As [the test we made in my fork](https://github.com/yin1999/translated-content/pull/24) would not get duplicate suggestions.

**Comment by zhanghuize-git** [comment]
> Hey @OnkarRuikar. reviewdog created many duplicate suggestions here. Can you take a look :)
> 
> I'm not sure if this is caused by [the temporary commit](https://github.com/mdn/translated-content/blob/d9b0bf1d36ca49aa42b9e88547ee86718b56d994/.github/workflows/pr-check-lint_content.yml#L36-L40).
> 
> As [the test we made in my fork](https://github.com/yin1999/translated-content/pull/24) (which has checkout the head sha) would not get duplicate suggestions.
你好，我再提交一个pr时，自动化工具提示有 markdown 有 lint 错误，但是我也没有看到相关的日志，可以得到你的帮助吗

File: files/zh-cn/web/api/window/afterprint_event/index.md [hunk 1]
```diff
@@ -1,33 +1,26 @@
 ---
-title: afterprint
+title: "Window: afterprint event"
```
Comment (yin1999) [inline files/zh-cn/web/api/window/afterprint_event/index.md:2]:
```suggestion
title: Window：afterprint 事件
```

File: files/zh-cn/web/api/window/afterprint_event/index.md [hunk 2]
```diff
@@ -1,33 +1,26 @@
...
+在像 {{domxref("EventTarget.addEventListener", "addEventListener()")}} 的方法中使用事件名称，或设置事件处理属性。
+
+```js
+addEventListener("afterprint", (event) => {});
+onafterprint = (event) => {};
+```
+
+## 事件类型
+
+一个通用的 {{domxref("Event")}}.
```
Comment (yin1999) [inline files/zh-cn/web/api/window/afterprint_event/index.md:23]:
```suggestion
一个通用的 {{domxref("Event")}}。
```

File: files/zh-cn/web/api/window/afterprint_event/index.md [hunk 3]
```diff
@@ -1,33 +1,26 @@
...
 
 ## 例子
```
Comment (yin1999) [inline files/zh-cn/web/api/window/afterprint_event/index.md:25]: 请参考：https://github.com/mdn/translated-content/blob/main/docs/zh-cn/glossary.md#%E6%A0%87%E9%A2%98%E8%A1%A8%E6%A0%BC

```suggestion
## 示例
```

File: files/zh-cn/web/guide/printing/index.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
+---
+title: 打印
+slug: Web/Guide/Printing
+---
+
+<section id="Quick_links">
+  {{ListSubpagesForSidebar("/en-US/docs/Web/Guide")}}
```
Comment (yin1999) [inline files/zh-cn/web/guide/printing/index.md:7]:
```suggestion
  {{ListSubpagesForSidebar("/zh-CN/docs/Web/Guide")}}
```

File: files/zh-cn/web/guide/printing/index.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+## 例子
+
+以下是一些常见示例。
+
+### 完成后打开并自动关闭弹出窗口
+
+如果你希望在用户打印完 [弹出窗口](/zh-CN/docs/Web/API/Window/open)（例如文档的打印机友好版本）的内容后自动关闭该窗口，你可以使用这样的代码：
+
+```html
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/web/guide/printing/index.md:57]: 请不要忽略 linter 给出的建议

File: files/zh-cn/web/guide/printing/index.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+      function popuponclick() {
+        const my_window = window.open(
+          "",
+          "mywindow",
+          "status=1,width=350,height=150",
+        );
+        my_window.document.write("<html><head><title>Print Me</title></head>");
+        my_window.document.write('<body onafterprint="self.close()">');
+        my_window.document.write(
+          "<p>When you print this window, it will close afterward.</p>",
```
Comment (yin1999) [inline files/zh-cn/web/guide/printing/index.md:73]: 建议对内容进行本地化

File: files/zh-cn/web/guide/printing/index.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+      }
+    </script>
+  </head>
+  <body>
+    <p>
+      To try out the <code>afterprint</code> event, click the link below to open
+      the window to print. You can also try changing the code to use
+      <code>beforeprint</code> to see the difference.
+    </p>
+    <p><a href="javascript: popuponclick()">Open Popup Window</a></p>
```
Comment (yin1999) [inline files/zh-cn/web/guide/printing/index.md:85]: 同

File: files/zh-cn/web/guide/printing/index.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+  </body>
+</html>
+```
+
+## 参见
+
+- [`window.print`](/zh-CN/docs/Web/API/Window/print)
+- {{ domxref("window.beforeprint_event", "beforeprint") }} 事件
+- {{ domxref("window.afterprint_event", "afterprint") }} 事件
+- [Media queries](/zh-CN/docs/Web/CSS/CSS_media_queries/Using_media_queries)
```
Comment (yin1999) [inline files/zh-cn/web/guide/printing/index.md:148]: 媒体查询

**Pull request closed without merging**